theme/palette/brand.less

```diff
@@ -5,6 +5,7 @@
 @sk-blue-light: #184da8;
 @sk-blue-lighter: #2765cf;
 @sk-pink: #e60278;
+@sk-pink-dark: #cc026a;
```

Comment:
I believe this is less of a brand colour and more of a darker critical variant as you've done with `positive` and `info`. What are your thoughts?

Reply:
Yes, this is not brand colour,  I put it here as I thought this was a variation on `@sk-pink` similar to `@sk-blue-dark`. What do you think about moving it to `elements` as `@sk-critical-dark` rather than `@sk-pink-dark`.

Reply:
Thats the point i was hoping to make, its not a variant on `sk-pink`, its a darker variant of `criticial`. The fact that critical is `sk-pink` is implementation detail.